core-server/src/main/java/org/orbisgis/orbisserver/control/web/IndexController.java

```diff
@@ -61,21 +70,34 @@
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.DataInputStream;
+import java.io.File;
 import java.io.InputStream;
 import java.util.List;
+import java.util.ArrayList;
 
 /**
- * Very light instance of DefaultController containing a WpsServer.
+ * Instance of DefaultController containing a WpsServer.
  *
  * @author Sylvain PALOMINOS
+ * @author Guillaume MANDE
  */
 @Controller
 public class IndexController extends DefaultController {
 
-    /**
-     * Gets the instance of the WpsServer.
-     */
+    /** Gets the instance of the WpsServer. */
     private WpsServer wpsServer = WpsServerManager.getWpsServer();
+    /** List all of the OrbisWPS processes into a String which be displayed on the index page. */
+    private String processesList = "";
+    /**  Used to display the corresponding xml file to GetCapabilities method. */
+    public WPSCapabilitiesType wpsCapabilitiesType = null;
```

Comment:
Do not make a variable be `public` unless it is a `static` one. If you need to make it accessible, use getters and setters.